scripts/run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Would you mind renaming this file to `server.sh` for conformity with other projects' standard script names? The README instructions may also need to be updated to match (Sorry!).